client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -219,11 +221,15 @@ class SimplePaymentsDialog extends Component {
 					.addPost( productToCustomPost( productForm ) )
 					.then( newProduct => {
 						dispatch( receiveUpdateProduct( siteId, customPostToProduct( newProduct ) ) );
+						analytics.tracks.recordEvent( 'calypso_simple_payments_button_create', {
+							price: productForm.price,
+							currency: productForm.currency,
```

Comment:
Can we add the product `ID` to the payload? and add it in 'calypso_simple_payments_button_insert' and 'calypso_simple_payments_button_delete' ? That should let us create button lifecycle funnels. 

Actually would that be a privacy risk?